src/components/Forms/CreateMemberForm/CreateMemberForm.jsx

```diff
@@ -73,7 +68,7 @@ export class CreateMemberForm extends React.Component {
     const { formErrors, isValid } = this.state;
 
     return (
-      <form onSubmit={this.handleSubmit} className={style.wrapper}>
+      <Form onSubmit={this.handleSubmit}>
```

Comment:
You don't need to use onSubmit on your `<Form/>`, if your button 'Save' handles this click. Also, dont throw event to your `handleSubmit`

Reply:
Yes, it was redundant. Thanks.